apps/site/src/app/admin/applicants/hackers/HackerApplicants.tsx

```diff
@@ -77,6 +78,11 @@ function HackerApplicants() {
 						header: "Status",
 						content: ApplicantStatus,
 					},
+					{
+						id: "reviewers",
+						header: "",
```

Comment:
Question: is there a particular reason you left the header empty?

Reply:
I didn't want to add a header because it didn't looked redundant and I couldn't find a text style that looked good with the labels that I added.

![image](https://github.com/user-attachments/assets/b8babf1a-e463-433b-91ae-b18231684144)